tests/queries/0_stateless/02961_drop_tables.sql

```diff
@@ -0,0 +1,25 @@
+DROP DATABASE IF EXISTS 02961_db1;
```

Comment:
CI runs tests in parallel. There is a chance that another test will delete `02961_db1` (l. 1), right after a test created it (l. 2).

As a solution, please don't create/drop databases at all, or mark the test `-- Tags: no-parallel`